src/reducer/resource/data.js

```diff
@@ -45,8 +45,10 @@ const addRecords = (newRecords = [], oldRecords) => {
     // remove outdated old records
     const latestValidDate = new Date();
     latestValidDate.setTime(latestValidDate.getTime() - cacheDuration);
-    const oldValidRecordIds = Object.keys(oldRecords.fetchedAt)
-        .filter(id => oldRecords.fetchedAt[id] > latestValidDate);
+    const oldValidRecordIds = oldRecords.fetchedAt
```

Comment:
This change allows me to import a serialized state (exported via the redux dev tools). This was required to test the patch.